tests/test_bundle_download.py

```diff
@@ -0,0 +1,167 @@
+                parser = ConfigParser()
+                parser.export_config_file(config=def_args, filepath=def_args_file)
+                cmd = ["coverage", "run", "-m", "monai.bundle", "download", "--args_file", def_args_file]
+                cmd += ["--url", url]
+                subprocess.check_call(cmd)
+                for file in bundle_files:
+                    file_path = os.path.join(tempdir, bundle_name, file)
+                    self.assertTrue(os.path.exists(file_path))
+                if file == "network.json":
+                    self.assertTrue(check_hash(filepath=file_path, val=hash_val))
+
+
+class TestLoad(unittest.TestCase):
+    @parameterized.expand([TEST_CASE_3])
+    @skip_if_quick
+    def test_load_weights(self, bundle_files, bundle_name, repo, device, model_file):
+        with skip_if_downloading_fails():
+            # download bundle, and load weights from the downloaded path
+            with tempfile.TemporaryDirectory() as tempdir:
+                # load weights
+                weights = load(
+                    name=bundle_name,
+                    bundle_dir=tempdir,
+                    repo=repo,
+                    progress=False,
+                    device=device,
+                    model_file=model_file,
+                )
+
+                # prepare network
+                with open(os.path.join(tempdir, bundle_name, bundle_files[2])) as f:
+                    net_args = json.load(f)["network_def"]
+                model_name = net_args["_target_"]
+                del net_args["_target_"]
+                model = nets.__dict__[model_name](**net_args)
+                model.to(device)
+                model.load_state_dict(weights)
+                model.eval()
+
+                # prepare data and test
+                input_tensor = torch.load(os.path.join(tempdir, bundle_name, bundle_files[4]), map_location=device)
+                output = model.forward(input_tensor)
+                expected_output = torch.load(os.path.join(tempdir, bundle_name, bundle_files[3]), map_location=device)
+                torch.testing.assert_allclose(output, expected_output)
+
+                # load instantiated model directly and test, since the bundle has been downloaded,
+                # there is no need to input `repo`
+                model_2 = load(
+                    name=bundle_name,
+                    bundle_dir=tempdir,
+                    progress=False,
+                    device=device,
+                    model_file=model_file,
+                    net_name=model_name,
+                    **net_args,
+                )
+                model_2.eval()
+                output_2 = model_2.forward(input_tensor)
+                torch.testing.assert_allclose(output_2, expected_output)
+
+    @parameterized.expand([TEST_CASE_4])
+    @skip_if_quick
+    def test_load_ts_module(self, bundle_files, bundle_name, repo, device, model_file):
```

Comment:
Could you please help add unit tests for the `config_files` arg? May also verify the loaded config content.

Thanks.